canonical/snapcraft.io
# Add larger favicon for google

## Done
- Add 144x144 favicon

## How to QA
- Visit https://snapcraft-io-4388.demos.haus/
- Ensure favicon still displays
